tests/tutorials/1_hello_world.cmake

```diff
@@ -12,11 +12,12 @@ include(cmake_test/cmake_test)
 #access to all of the CMakeTest machinery. The testing philosophy of CMakeTest
 #closely follows that of the Catch2 C++ unit testing framework. In particular
 #this means each of your unit tests must be contained within an ``add_test``
-#block. The block starts with ``ct_add_test(<test_name>)`` and ends with
-#``ct_end_test()`` (all CMakeTest functions are namespaced with a prefix ``ct_``
+#block. The block starts with :obj:`~cmake_test/add_test.ct_add_test`
+#(all CMakeTest functions are namespaced with a prefix ``ct_``
 #to mitigate against collisions with similarly named functions from other
-#modules). ``ct_add_test`` and ``ct_end_test`` define a block scope. The CMake
-#code in this block scope will serve as the contents of the unit test.
```

Comment:
Originally I had a script which stripped these tutorial comments out and created an `.rst` file from it. This was before I knew of `.. literalinclude`. Point being, feel free to put these doc comments into a `.rst` file and then `.. literalinclude::` the test.